tests/install_upgrade_operators/role_aggregation/conftest.py

```
@@ -0,0 +1,86 @@
+def edit_role_binding(admin_client, namespace):
+    """RoleBinding granting edit ClusterRole to the unprivileged user."""
+    yield from unprivileged_role_binding(admin_client=admin_client, namespace_name=namespace.name, role_name="edit")
+
+
+@pytest.fixture(scope="class")
+def view_role_binding(admin_client, namespace):
+    """RoleBinding granting view ClusterRole to the unprivileged user."""
+    yield from unprivileged_role_binding(admin_client=admin_client, namespace_name=namespace.name, role_name="view")
+
+
+@pytest.fixture(scope="function")
```

Ruff (0.16.1)

[warning] `scope='function'` is implied in `@pytest.fixture()`

Remove implied `scope` argument

(PT003)